src/renderer/pages/Apps.tsx

```diff
@@ -136,6 +138,17 @@ export default function Apps() {
     const appSettings =
       await window.electron.ipcRenderer.madaraApp.getAppSettings(appId);
     const appConfig = APPS_CONFIG.apps.find((app) => app.id === appId);
+    const requiresRunningNode = true;
```

Comment:
I guess you hardcoded this for testing? It can probably be removed now. You can add this property to all apps inside `app.ts` folder inside configs.

Reply:
hi, actually it is part of the code that i developed (not only for testing), but yes i will move it to app.ts, thanks!